scripts/install-worker.sh

```diff
@@ -151,11 +168,16 @@ fi
 ### Containerd setup ##########################################################
 ###############################################################################
 
+# install containerd and lock version
+# if docker is installed as well, this will run twice, but it's idempotent
+sudo yum install -y containerd-${CONTAINERD_VERSION}
+sudo yum versionlock containerd-*
```

Comment:
any downside to moving the `runc` installation here too, and removing this: https://github.com/awslabs/amazon-eks-ami/pull/938/files#diff-dff0aa3a99c138c812c9364e64a6f95abec4590a27f0fb2b448231342b4a3b89R143-R149 ?

Reply:
I don't think so, just need to move on the containerd config above the docker install since docker depends on containerd and runc

Reply:
yep, sounds good. In 1.24, we're disallowing docker as the runtime but leaving the packages installed; but this will lay the groundwork for us to remove it altogether in 1.25.

Reply:
Moved and tested w/ both containerd and dockerd args to bootstrap script with caching enabled in the AMI and caching disabled in the AMI.